nodejs.spec

```diff
@@ -307,6 +307,8 @@ NODE_PATH=%{buildroot}%{_prefix}/lib/node_modules %{buildroot}/%{_bindir}/node -
 %{_pkgdocdir}/npm/doc
 
 %changelog
+* Thu May 22 2019 Lucas Holmquist <lholmqui@redhat.com> - 12.3.0-1
```

Comment:
I think this will cause an error. It should be `Wed` instead of `Thu` right?

Reply:
ffs :)